packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -601,7 +606,8 @@ export default class Calendar extends React.Component {
             utcOffset={this.props.utcOffset}
             handleCalendarKeyDown={this.props.handleCalendarKeyDown}
             locale={this.props.locale}
-            intl={this.props.intl} />
+            intl={this.props.intl}
+            disableKeyboardSelected={this.state.disableKeyboardSelected} />
```

Comment:
When is this ever set to false after the initial render?

Reply:
Resolved in 15987a0